lldb/tools/lldb-dap/DAP.h

```diff
@@ -416,6 +416,14 @@ struct DAP {
 
   lldb::SBMutex GetAPIMutex() const { return target.GetAPIMutex(); }
 
+  /// Sends an error response in DAP with success=false and an error message.
+  ///
+  /// \param[out] response
+  ///   The response object to update.
+  /// \param[in] message
+  ///   The error message.
+  void SendErrorResponse(llvm::json::Object &response, llvm::StringRef message);
+
```

Comment:
Could we move this helper to the LegacyRequestHandler? I think with the new `RequestHandler<>` type we shouldn't need this and having it on the DAP object may add new usages of this before we can finish migrating the request handlers to the new `RequestHandler<>` type.